### ISSUE TITLE
Broken Dependencies using Fedora COPR Install Method

### ISSUE THREAD

**Issue by Loklann** [luisbocanegra/kde-material-you-colors#207]
**Describe the bug**
When trying to install `kde-material-you-colors` via the COPR repo, as per the provided instructions in the README, `dnf` can't find `python-pywal` which is a weak dependency for `kde-material-you-colors`, and the installation fails. Installing `pywal` using `pip` / `pipx` doesn't resolve this issue.
See :
```

 Error: 
  Problem: conflicting requests
   - nothing provides python-pywal needed by kde-material-you-colors-1.9.1-1.git.3.ff01ace.fc40.x86_64 from copr:copr.fedorainfracloud.org:luisbocanegra:kde-material-you-colors


```

**To Reproduce**
Try to install via the COPR repo for Fedora.

**Expected behavior**
The package should install properly.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - Operating System: Fedora 40
 - KDE Plasma Version: 6.0.4
 - KDE Frameworks Version: 6.2.0
 - Qt Version: 6.7.0
 - Graphics Platform: Wayland
 - Things in your system that might be relevant: N/A

**Additional context**
N/A

**Comment by luisbocanegra** [comment]
Yeah I built it manually then forgot I did, then mistakenly thought it was an official package after searching (`dnf search pywal`) for it to make sure I didn't need to provide it... Turns out that command returns installed packages too. Will add it to the repository ASAP.

Thanks for letting me know.

**Comment by luisbocanegra** [comment]
Fixed

**Comment by Loklann** [comment]
Great ! Also, this app is amazing, I would not use KDE without it, you're doing an amazing job!
